iidm/iidm-xml-converter/src/main/java/com/powsybl/iidm/xml/ThreeWindingsTransformerXml.java

```diff
@@ -97,20 +122,27 @@ protected ThreeWindingsTransformer readRootElementAttributes(ThreeWindingsTransf
         double ratedU1 = XmlUtil.readDoubleAttribute(context.getReader(), "ratedU1");
         double r2 = XmlUtil.readDoubleAttribute(context.getReader(), "r2");
         double x2 = XmlUtil.readDoubleAttribute(context.getReader(), "x2");
+        double g2 = XmlUtil.readOptionalDoubleAttribute(context.getReader(), "g2", 0.0);
+        double b2 = XmlUtil.readOptionalDoubleAttribute(context.getReader(), "b2", 0.0);
         double ratedU2 = XmlUtil.readDoubleAttribute(context.getReader(), "ratedU2");
         double r3 = XmlUtil.readDoubleAttribute(context.getReader(), "r3");
         double x3 = XmlUtil.readDoubleAttribute(context.getReader(), "x3");
+        double g3 = XmlUtil.readOptionalDoubleAttribute(context.getReader(), "g3", 0.0);
+        double b3 = XmlUtil.readOptionalDoubleAttribute(context.getReader(), "b3", 0.0);
```

Comment:
Why are these attributes optional? I would rather have them required and clearly break the compatibility.

Reply:
Discarded